app/views/student/knowledges/index.html.haml

```diff
@@ -0,0 +1,3 @@
+= @knowledges.each do |knowledge|
+  %div
+    = link_to "#{knowledge.title}", mentor_knowledge_path(knowledge)
```

Comment:
__[HAML-Lint]__

Files should end with a trailing newline


<!-- Comment by [SideCI](https://sideci.com) -->